### DISCUSSION TITLE
Gather gifs, stickers and emoji in one place

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [element-hq/element-meta#1529] [Ideas]
![IMG_20230428_115133](https://user-images.githubusercontent.com/98482469/235182324-b5515e35-e9ff-42c5-ad63-fedc8f03f797.jpg)
## Telegram
![Screenshot_20230428-115037_Telegram-cropped](https://user-images.githubusercontent.com/98482469/235182672-bb045890-d314-48d0-9d14-b3817af55fb5.jpg)
## Whatsapp
![Screenshot_20230428-120420_WhatsApp-cropped](https://user-images.githubusercontent.com/98482469/235184042-6909c6ad-01f6-449e-8a7a-9dd18f261662.jpg)

### this is much more intuitive and natural than separating the stickers and gifs on the "+" button

**Comment by Dom23565** [comment]
`
```c++

``` `
